engine/src/main/java/org/hibernate/search/engine/spi/AbstractDocumentBuilder.java

```diff
@@ -152,12 +152,12 @@ public void postInitialize(IndexedTypeSet indexedClassesSet) {
 	}
 
 	/**
-	 * If we have a work instance we have to check whether the instance to be indexed is contained in any other indexed entities.
-	 *
 	 * @see #appendContainedInWorkForInstance(Object, WorkPlan, ContainedInRecursionContext, String)
 	 * @param instance the instance to be indexed
 	 * @param workPlan the current work plan
 	 * @param currentRecursionContext the current {@link org.hibernate.search.engine.spi.ContainedInRecursionContext} object used to check the graph traversal
+	 * @deprecated Use {@link #appendContainedInWorkForInstance(Object, WorkPlan, ContainedInRecursionContext, String)}
+	 * instead, and pass the appropriate tenant ID.
 	 */
 	public void appendContainedInWorkForInstance(Object instance, WorkPlan workPlan, ContainedInRecursionContext currentRecursionContext) {
```

Comment:
Should we add this method to the ones we don't want the codebase to use?

Reply:
Good idea, done.